models/iterable__events.sql

```diff
@@ -1,6 +1,6 @@
 {{ config(
         materialized='incremental',
-        unique_key=['unique_event_id'],
+        unique_key=['unique_event_id','created_at'],
```

Comment:
Based on our internal discussion, this change can be reverted. We just will need to make sure we update the seed data to replicate a realistic environment.

Reply:
updated and updated seed!